README.md

```diff
@@ -18,15 +18,104 @@ npm install -g @gridgain/thin-client
 
 If you want to install the Thin Client library from GridGain sources, please follow the steps:
 
-1. Download GridGain sources to `local_gridgain_path`
-2. Go to `local_gridgain_path/modules/platforms/nodejs` folder
+1. Download Node.js GridGain client sources to `nodejs-thin-client`
+2. Go to `nodejs-thin-client` folder
 3. Execute `npm link` command
 4. Execute `npm link @gridgain/thin-client` command (needed only for examples)
 
 ```bash
-cd local_gridgain_path/modules/platforms/nodejs
+cd nodejs-thin-client
 npm link
 npm link @gridgain/thin-client #linking examples (optional)
 ```
+---------------------------------------------------------------------
 
-For more information, see [GridGain Node.JS Thin Client documentation](https://apacheignite.readme.io/docs/nodejs-thin-client).
\ No newline at end of file
+# Tests #
+
+NodeJS Client for GridGain contains [Jasmine](https://www.npmjs.com/package/jasmine) tests to check the behavior of the client. the tests include:
+- functional tests which cover all API methods of the client
+- examples executors which run all examples except AuthTlsExample
+- AuthTlsExample executor
+
+## Tests Installation ##
+
+Tests are installed along with the client.
+Follow the [Installation instructions](#installation).
+
+## Tests Running ##
+
+1. Run GridGain server locally or remotely with default configuration.
+2. Set the environment variable:
+    - **APACHE_IGNITE_CLIENT_ENDPOINTS** - comma separated list of GridGain node endpoints.
+    - **APACHE_IGNITE_CLIENT_DEBUG** - (optional) if *true*, tests will display additional output (default: *false*).
+3. Alternatively, instead of the environment variables setting, you can directly specify the values of the corresponding variables in [nodejs-thin-client/spec/config.js](./spec/config.js) file.
+4. Run the tests:
+
+### Run Functional Tests ###
+
+Call `npm test` command from `nodejs-thin-client` folder.
+
+### Run Examples Executors ###
+
+Call `npm run test:examples` command from `nodejs-thin-client` folder.
+
+### Run AuthTlsExample Executor ###
+
+It requires running GridGain server with non-default configuration (authentication and TLS switched on).
```

Comment:
Language: `It requires running GridGain server` -> `GridGain server node is required`

Reply:
Activated GridGain node?